### ISSUE TITLE
Adding SCANsat to CactEye Optics mod

### ISSUE THREAD

**Issue by linuxgurugamer** [KSPModStewards/SCANsat#400]
I'm planning on adding compatibilty with SCANsat to the CactEye Optics mod I'm in the process of adopting.

Would this be all that's needed (there are two parts):

`@PART[tele_body]:NEEDS[SCANsat]
{
	MODULE
	{
		name = ModuleSCANresourceScanner
		sensorType = 128
		fov = 3
		min_alt = 15000
		max_alt = 7500000
		best_alt = 500000
		scanName = Resource Scan
		RESOURCE
		{
			name = ElectricCharge
			rate = 0.5
		}
	}
}
@PART[tele_body2]:NEEDS[SCANsat]
{
	MODULE
	{
		name = ModuleSCANresourceScanner
		sensorType = 256
		fov = 2
		min_alt = 10000
		max_alt = 500000
		best_alt = 150000
		scanName = Ore Scan
		RESOURCE
		{
			name = ElectricCharge
			rate = 2
		}
	}
	
	MODULE
	{
		name = SCANresourceDisplay
		ResourceName = Ore
	}
}
`

**Comment by DMagic1** [comment]
That should work.

Do these parts also have the stock resource scanner modules? If so the ModuleSCANresourceScanner can optionally disable them. If not then you can use the regular SCANsat part module.

The fov value is field of view. It is basically the number of whole-number degrees that will be scanned at the ideal altitude around your current position, but normalized to the body's radius in relation to the home body radius.

So around Kerbin, with an fov of 2 and scanning at the ideal altitude you will scan a 4x4 degree box around your vessel (at the equator, the horizontal scanning width is adjusted by latitude). Around Minmus it will be a much larger number of degrees (you can set the fov value lower than 1 to get smaller scanning width around small bodies), and around Jool much smaller (there is a lower limit of 1 degree and an upper limit of 20).

So kind of complicated. Basically just play around with it. I usually stick to the 1-3 range.

**Comment by linuxgurugamer** [comment]
No, these parts do not have the stock resource scanner module, although maybe I should add it to the mod.

And thank you for such a wonderful mod.

**Comment by linuxgurugamer** [comment]
I'm not going to add the stock resource scanner, since that would be opposite the idea behind the mod of science needing time.
Thanks for your replies.

**Comment by linuxgurugamer** [comment]
Just curious as to your feeling about these:

CactEye mod has two scopes, the Cacti and the Fungi (smaller).  What I'm going to do is to have the FOV for the Cacti to be 4 and the FOV for the Fungi to be 2, but to have the altitudes for the Cacti to be much higher than the Fungi.
Also, the Fungi will have a much higher EC usage.
Here are the patches I'm going to use, would appreciate your taking a quick look at them:

`
@PART[tele_body]:NEEDS[SCANsat]
{
	MODULE
	{
		name = ModuleSCANresourceScanner
		sensorType = 256 //2^8
		fov = 4
		min_alt = 20000
		max_alt = 7500000
		best_alt = 500000
		scanName = Resource Scan
		RESOURCE
		{
			name = ElectricCharge
			rate = 0.5
		}
	}
	MODULE
	{
		name = SCANresourceDisplay
		ResourceName = Ore
	}

	MODULE
	{
		name = ModuleKerbNetAccess
		EnhancedSituationMask = 48
		EnhancedMinimumFoV = 13
		EnhancedMaximumFoV = 72
		MinimumFoV = 13
		MaximumFoV = 72
		AnomalyDetection = 0
		RequiresAnimation = False
		DISPLAY_MODES
		{
			Mode = Resources,#autoLOC_900321 //#autoLOC_900321 = Resources
		}
	}


	MODULE
	{
		name = SCANexperiment
		experimentType	= SCANsatResources
	}
}
@PART[tele_body2]:NEEDS[SCANsat]
{
	MODULE
	{
		name = ModuleSCANresourceScanner
		sensorType = 256 //2^8
		fov = 2
		min_alt = 10000
		max_alt = 500000
		best_alt = 150000
		scanName = Ore Scan
		RESOURCE
		{
			name = ElectricCharge
			rate = 2
		}
	}
	
	MODULE
	{
		name = SCANresourceDisplay
		ResourceName = Ore
	}

	MODULE
	{
		name = ModuleKerbNetAccess
		EnhancedSituationMask = 48
		EnhancedMinimumFoV = 13
		EnhancedMaximumFoV = 72
		MinimumFoV = 13
		MaximumFoV = 72
		AnomalyDetection = 0
		RequiresAnimation = False
		DISPLAY_MODES
		{
			Mode = Resources,#autoLOC_900321 //#autoLOC_900321 = Resources
		}
	}

	MODULE
	{
		name = SCANexperiment
		experimentType	= SCANsatResources
	}
}

`

**Comment by linuxgurugamer** [comment]
I'm thinking of going a different route.  Rather than integrate it into the telescope, I am thinking of disabling the surface attachment on the scope and adding several attach nodes instead.  I'm going to expand on this idea in the thread to see what people think.
